dataactvalidator/interfaces/interfaceHolder.py

```diff
@@ -8,10 +9,17 @@ class InterfaceHolder:
 
     def __init__(self):
         """ Create the interfaces """
-        self.jobDb = ValidatorJobTrackerInterface()
-        self.errorDb = ValidatorErrorInterface()
-        self.stagingDb = ValidatorStagingInterface()
-        self.validationDb = ValidatorValidationInterface()
+        if BaseInterface.interfaces is None:
+            self.jobDb = ValidatorJobTrackerInterface()
+            self.errorDb = ValidatorErrorInterface()
+            self.stagingDb = ValidatorStagingInterface()
+            self.validationDb = ValidatorValidationInterface()
+            BaseInterface.interfaces = self
```

Comment:
I don't know the class diagram well enough to comment on the proposed structure, but I'd recommend something dependency-injection-y. Maybe have a global variable which gets set on validator (and broker) init and refer to that here (please excuse the terrible variable names):

```python
# dataactbroker/app.py
from dataactcore.models.baseInterface import BaseInterface
from dataactbroker.handlers import InterfaceHolder
BaseInterface.interface =  InterfaceHolder

# dataactvalidator/app.py
from dataactcore.models.baseInterface import BaseInterface
from dataactvalidator.interfaces import InterfaceHolder
BaseInterface.interface =  InterfaceHolder
```

My concern with the current design revolves around a constructor mutating global state. As it stands, we're creating an implicit coupling between the two InterfaceHolders such that the order of their creation affects how they'll function. We _know_ that we shouldn't create them at the same time, but it's not explicit. I'd argue that setting the values in the init scripts is a step in the right direction.

That said, I don't think it should be a blocker if we're okay with that tech debt.